package.json

```diff
@@ -82,60 +91,29 @@
     ]
   },
   "devDependencies": {
-    "@babel/core": "~7.3.4",
+    "@babel/core": "~7.12.1",
+    "@babel/preset-env": "~7.12.1",
+    "@babel/preset-react": "~7.12.1",
+    "@babel/cli": "~7.12.1",
+    "@babel/register": "~7.12.1",
```

Comment:
Looks like `babel`'s setup change quite a bit? Any documentation or articles you followed to update these?

Reply:
Babel 7 has some notable changes like they packages started with `@` instead of `babel-core`. I saw a mix of both Babel 6 and 7. This is good to start with https://babeljs.io/docs/en/v7-migration. I read little more articles. Just followed build, run, test and delete approach.

Reply:
Thanks for the info!